tools/docker-network/docker-compose.yml

```diff
@@ -47,6 +47,7 @@ services:
       --valueLayer.snapshot.file=/tmp/assets/7R1itJx5hVuo9w9hjg5cwKFmek4HMSoBDgJZN8hKGxih.bin
       --syncbeacon.broadcastInterval=5
       --syncbeacon.startSynced=true
+      --statement.writeStatement=true
```

Comment:
Don't want to replica's to write statements?

Reply:
Yes, that was the idea. Should we let everybody do it?

Reply:
I guess it's fine if it's only the master. Was just wondering whether it was on purpose.